recipe/meta.yaml

```diff
@@ -36,6 +31,7 @@ requirements:
     - nccl
     - libfaiss =*=*_cuda
     - spdlog =1.7.0
```

Comment:
Is the `spdlog` dependency satisfied by `librmm` now?  (I originally added it specifically for RMM prior to the separate `librmm` package).

Reply:
Does `libcugraph` make any direct usage of `spdlog`? If not, agree this can be dropped

Reply:
> Does `libcugraph` make any direct usage of `spdlog`? If not, agree this can be dropped

`libcugraph` does not use `spdlog` directly. [Here's](https://github.com/rapidsai/cugraph/blob/branch-0.20/conda/recipes/libcugraph/meta.yaml) the RAPIDS `libcugraph` recipe for reference.

Reply:
Good catch! CUDA 10.2 builds succeeded, so looks like this works. I'll drop `spdlog` momentarily.